utils/S3UtilsMongoClient.js

```diff
@@ -44,13 +46,13 @@ class S3UtilsMongoClient extends MongoClientInterface {
 
             const locationConfig = getLocationConfig(log);
 
-            const usersBucketCreationDatesArray = await this.getCollection(USERSBUCKET).find({}, {
+            const cursorUsersBucketCreationDates = await this.getCollection(USERSBUCKET).find({}, {
```

Comment:
nit: `usersBucketCreationDatesCursor`

```suggestion
            const usersBucketCreationDatesCursor = await this.getCollection(USERSBUCKET).find({}, {
```

Reply:
I updated, even if this is indeed not stricly necessary, as we convert the cursor to an array (also applies to `forEach`s that complete); we now explicitly close the cursor. 
https://github.com/scality/s3utils/pull/310/files#diff-3a778b361ddfba6106347ff0cb053bcbb1a84374fa0d7e6e6413e7c6fb6ca5cdR54